index.html

```diff
@@ -467,7 +466,8 @@ <h3>
                     some countries postal codes are so fine-grained that they
                     can uniquely identify a recipient.
                   </p>
-                </div>Let <var>redactList</var> be the empty list. Optionally,
+                </div>  
```

Comment:
Please revert this one. I know it looks weird, but the markup is ok 👍

Reply:
Reverted!